api/v1beta1/types.go

```diff
@@ -147,6 +147,15 @@ type NetworkSpec struct {
 	// +kubebuilder:default:=1460
 	// +optional
 	Mtu int64 `json:"mtu,omitempty"`
+
+	// MinPortsPerVM: Minimum number of ports allocated to a VM from this NAT
+	// config. If not set, a default number of ports is allocated to a VM. This is
+	// rounded up to the nearest power of 2. For example, if the value of this
+	// field is 50, at least 64 ports are allocated to a VM.
+	// +kubebuilder:validation:Minimum:=64
```

Comment:
@barbacbd Do we need to set this minimum value?
Wouldn't `0` for example be a valid number for the internal GCP platform validation, as thats `2^0`?

Reply:
Why would it increase to 64? If by definition 1 is POW 2, that should be ok right?
Might be worth trying. If GCP errors with less than 64, then we can set the Minimum validation like you did :)

Reply:
TY